dp/cloud/python/magma/configuration_controller/response_processor/strategies/response_processing.py

```diff
@@ -91,7 +91,10 @@ def process_spectrum_inquiry_response(obj: ResponseDBProcessor, response: DBResp
         session: Database session
     """
 
-    if response.response_code == ResponseCodes.DEREGISTER.value:
+    if any([
```

Comment:
It looks like this part is duplicated in multiple functions (it appears at the beginning of almost all requests) can we extract this to a function or maybe even some decorator?